doc/releases/release-notes-3.6.rst

```diff
@@ -1229,10 +1229,26 @@ Libraries / Subsystems
 
 * ZBus
 
+  * Replace mutexes with semaphores to lock channels and implement the Highest Locker Protocol (HLP)
+    priority boost for the zbus operations. This feature avoids priority inversions and preemptions,
+    making the VDED delivery process faster and more consistent. More details at
+    https://github.com/zephyrproject-rtos/zephyr/pull/63183.
```

Comment:
This needs to be a proper reference, see examples in the migration guide.

Reply:
Done. I have added the reference to the other items as well. Thanks @henrikbrixandersen.